Makefile.toml

```diff
@@ -820,15 +822,20 @@ done
 # modules for a given release.
 LINK_REPO_TARGETS=("--link-target ${BUILDSYS_KMOD_KIT_PATH}")
 
-# Include the root and data disk images in the repo if they exist
+# Include the root and data disk images in the repo with a friendly name
+# if they exist
 os_disk_img="${BUILDSYS_OUTPUT_DIR}/${BUILDSYS_NAME_FULL}.img.lz4"
+os_disk_img_friendly="${BUILDSYS_OUTPUT_DIR}/${BUILDSYS_NAME_FRIENDLY}.img.lz4"
 if [ -s "${os_disk_img}" ] ; then
-   LINK_REPO_TARGETS+=("--link-target ${os_disk_img}")
+   ln -snf "${os_disk_img}" "${os_disk_img_friendly}"
+   LINK_REPO_TARGETS+=("--link-target ${os_disk_img_friendly}")
```

Comment:
I'd prefer to do the linking in the `link-variant` step, and also clean up the links in the `link-clean` target.